pkg/consensus/consensus_test.go

```diff
@@ -65,8 +68,8 @@ func TestService_GetVerified(t *testing.T) {
 
 	t.Run("new transactions will be proposed in case of failure", func(t *testing.T) {
 		txx := srv.getVerifiedTx(10)
-		require.Equal(t, 2, len(txx), "there is only 1 tx in mempool")
-		require.Equal(t, txs[3], txx[1])
+		require.Equal(t, 1, len(txx), "there is no tx in mempool")
```

Comment:
There still should be one transaction in the pool and you're getting it. It's just that there is no miner tx added any more.